specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2020-07-01/bms.json

```diff
@@ -0,0 +1,479 @@
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Trigger Data Move": {
+            "$ref": "./examples/BackupDataMove/TriggerDataMove_Post.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NewErrorResponse": {
+      "properties": {
+        "error": {
+          "type": "object",
+          "description": "The error object.",
+          "properties": {
+            "code": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The error code."
+            },
+            "message": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The error message."
+            },
+            "target": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The error target."
+            },
+            "details": {
+              "readOnly": true,
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/NewErrorResponse"
+              },
+              "description": "The error details."
+            },
+            "additionalInfo": {
+              "readOnly": true,
+              "type": "array",
+              "items": {
+                "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ErrorAdditionalInfo"
+              },
+              "description": "The error additional info."
+            }
+          }
+        }
+      },
+      "description": "The resource management error response."
+    },
+    "PrepareDataMoveRequest": {
+      "description": "Prepare DataMove Request",
+      "type": "object",
+      "properties": {
+        "targetResourceId": {
+          "description": "ARM Id of target vault",
+          "type": "string"
+        },
+        "targetRegion": {
+          "description": "Target Region",
+          "type": "string"
+        },
+        "targetCloud": {
+          "description": "Target Cloud",
+          "type": "string"
+        },
+        "dataMoveLevel": {
+          "description": "DataMove Level",
+          "enum": [
+            "Invalid",
+            "Vault",
+            "Container"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "DataMoveLevel",
+            "modelAsString": true
+          }
+        },
+        "sourceContainerArmIds": {
+          "description": "Source Container ArmIds",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "ignoreMoved": {
+          "description": "Ignore the artifacts which are already moved.",
+          "type": "boolean"
+        }
+      }
+    },
+    "PrepareDataMoveRequestResource": {
+      "description": "Prepare DataMove Request",
+      "allOf": [
+        {
+          "$ref": "../2019-06-15/bms.json#/definitions/Resource"
```

Comment:
>    "$ref": "../2019-06-15/bms.json#/definitions/Resource" [](start = 6, length = 58)

Is this a full resource? Unless you are modeling as a resource you don't need id and other attributes for  this request object.